### DISCUSSION TITLE
Brave Rewards

### DISCUSSION THREAD

**Discussion by SteMoly** [Kunzisoft/KeePassDX#1041] [General]
Hi,
I would like to send you a tip through my Brave Browser but you are not verified, can this be sorted?
Sorry if this isn't the right place for this type of question as new to github so a point in the right direction.

**Comment by J-Jamet** [comment]
Hello,
This is absolutely the right place to ask the question, no worries about that. ;)

I don't really like brave rewards for several reasons:
- The brave browser rewards are not real BAT, they are only browser tokens that are not on the decentralized network.
- It is not possible to retrieve real BAT or convert Brave rewards without going through a centralized platform (Uphold). And I don't want to be forced to use a specific centralized platform (so without competition and not decentralized) to get real BAT (which are only an ethereum token)
- Cryptocurrencies are normally created to be interoperable and decentralized, but the Brave ecosystem is closed and does not allow the use of DeX directly from rewards, BAT is basically only an ERC20 ETH token, so a cold ethereum address would be logically sufficient, but this is currently not possible from rewards.

I had already linked the keepassdx site to brave to receive rewards, but actually I don't use this browser and I'm not going to register on uphold. So you can give them, but my only move afterwards is to distribute this rewards to other brave creators.

**Reply by SteMoly** [reply]
Thanks for the reply.
Looks like I've got a bit of researching to do on this and crypto.
Just thought I might try to pass them your way as a contribution to what I think is a great app.
